packages/amplify-graphiql-explorer/src/ClearDataModal.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { Button, Modal } from "semantic-ui-react";
+
+type Props = {
+  onClose: Function;
+  onClear: Function;
+};
```

Comment:
Could you add a comment about the intention for these callbacks? It's not obvious to me what the difference between these should be, since you're calling them both on reset, for example.